content/issues/tracking-your-work-with-issues/creating-an-issue.md

```diff
@@ -97,6 +97,28 @@ You can open a new issue from a specific line or lines of code in a file or pull
 {% data reusables.repositories.assign-an-issue-as-project-maintainer %}
 {% data reusables.repositories.submit-new-issue %}
 
+{% ifversion fpt %}
+
+## Creating an issue from discussion
+
+You can even create an issue from a discussion. Anyone with read access to the repository can create an issue from a discussion. Follow the steps below to create an issue from discussion :
```

Comment:
```suggestion
People with triage permission to a repository can create an issue from a discussion.

When you create an issue from a discussion, the contents of the discussion post will be automatically included in the issue body, and any labels will be retained. Creating an issue from a discussion does not convert the discussion to an issue or delete the existing discussion. For more information about {% data variables.product.prodname_discussions %}, see "[About discussions](/discussions/collaborating-with-your-community-using-discussions/about-discussions)."
```

Reply:
@lecoursen I think even people with read access can do so.

See even I can do that in github/docs :

![Create Issue from Discussion](https://user-images.githubusercontent.com/51746608/130724481-3ffdd41e-c5a5-491c-a480-dd2cc37df4f1.jpg)

I guess read access ≠ triage access

Reply:
Thanks! Our internal issue for this feature specifically mentions triage. Maybe we misunderstood, though! I'll double check with that team. 👍🏻

Reply:
@lecoursen I will take these in after you've confirmed with the team